src/release/issue-triaging.md

```diff
@@ -1,125 +1,276 @@
-- [`P-*`]: Priority labels. Applied using the [compiler prioritization procedure](../compiler/prioritization.md).
-- [`S-*`]: The status of an issue, for example `S-needs-repro`.
-- [`E-*`]: Calls for participation[^3], for example to minimize an issue.
-    - `E-mentor`: A mentor is available to help with the issue, which makes for good first issues.
-    - `E-needs-mcve`: This issue has a reproduction, but it is not minimal, it should be minimized.
-    - `E-needs-bisection`: This issue needs a bisection, for example using [cargo-bisect-rustc](https://github.com/rust-lang/cargo-bisect-rustc).
-    - `E-needs-investigation`: This issue needs further investigation to determine root causes and the nature of the issue.
-    - `E-needs-design`: This issue will require some substantial design effort (exploration, prototyping, discussions, etc.).
-    - `E-needs-test`: The issue has been fixed, but no test has been added for it. After someone adds a test, it can be closed.
-    - `E-{easy,medium,hard}`: Someone has estimated how hard the issue is to fix. This can help with finding good first issues, but is [bound to be inaccurate](https://en.wikipedia.org/wiki/Curse_of_knowledge).
+- Make sure the issue is tagged with `I-ICE`, `T-compiler` and `C-bug`.
+- Check that the issue is actually an ICE, and not more accurately described
+  with `I-crash` or `I-hang`.
+- If it is an older (like latest stable) version of rust, ask:
+    - Does the ICE reproduce on beta and latest nightly.
+    - If older stable, ask if the ICE reproduces on latest stable.
+- Check for duplicates, but don't close as duplicate unless you're sure they
+  represent the same underlying issue. Prefer simply linking to the issue as
+  possibly related/duplicate.
+    - A good way is to search the ICE message. Be careful to not close as
+      duplicate when the same ICE message may have different root causes.
+- If it does not have a reproduction, comment asking for one and add
+ ` S-needs-repro`.
+    - If there isn't one for around a month it should generally be closed.
+- If the reproduction is not minimal, add `E-needs-mcve` or create a MCVE
+  yourself. See the Initial triaging section for more details on MCVE procedure.
+- Add `A-*` labels based on the code that causes the issue (check backtraces!),
+  and the nature of the repro (eg. if the repro is a weird trait impl or the
+  backtrace points to `rustc_trait_selection`, add `A-traits`)
+- Add `T-*`, `WG-*`, `PG-*`, `F-*`, `requires-*`, and `regression-*` labels as
+  appropriate.
+    - Usually the ICE should be tagged `T-compiler`. If the issue concerns the
+      type system (e.g. trait solver), also tag the issue with `T-types`.
+
+## Further triaging
 
-See also section [Issue Triage](https://rustc-dev-guide.rust-lang.org/contributing.html#issue-triage) in the Rust Compiler Development Guide.
```

Comment:
Deleted this, I wanted to only keep label docs and stuff in one place, and forge docs seemed like a better fix. I'll send a r-d-g side PR to backlink to this page.

Reply:
https://github.com/rust-lang/rustc-dev-guide/pull/2194